pkg/graveler/ref/manager.go

```diff
@@ -547,11 +546,11 @@ func newCache(cfg CacheConfig) cache.Cache {
 }
 
 func (m *Manager) SetAddressToken(ctx context.Context, repository *graveler.RepositoryRecord, token string) error {
-	a := &graveler.AddressData{
+	a := &graveler.LinkAddressData{
 		Address:   token,
 		ExpiredAt: timestamppb.New(time.Now().Add(AddressTokenTime)),
```

Comment:
Why do we still need the ExpiredAt if it's embedded in the address?

Reply:
deleted